projects/aca-testing-shared/package.json

```diff
@@ -1,7 +1,13 @@
 {
-  "name": "aca-testing-shared",
+  "name": "@alfresco/aca-testing-shared",
```

Comment:
not sure if we need to publish this library to NPM

Reply:
we are using it in `apps`

<img width="456" alt="Screenshot 2023-01-09 at 09 56 00" src="https://user-images.githubusercontent.com/13885344/211271194-1073bc76-1412-443f-9499-58c9f6eed734.png">

Reply:
I've checked the apps and it's mind-blowing, someone imports a class from this lib but not uses it, we should probably consider removing this dependency from the apps and not publishing this lib to the NPM